JS-101: JavaScript Basics/js/booleans.js

```diff
@@ -0,0 +1,12 @@
+let correctGuess = false;
+const number = 6;
+const guess = prompt("Guess a number between 1 and 10");
+
+if (number === +guess) {
+  correctGuess = true;
+}
+if (correctGuess === true) {
```

Comment:
No need to explicitly compare it to `=== true`here. If `correctGuess` is truthy, then the block will be run.